rollup.config.js

```diff
@@ -18,13 +18,16 @@ const geckoBuild = {
       mainFields: ["module", "main", "browser"],
       preferBuiltins: true,
     }),
-    typescript({ include: ["*.ts+(|x)", "**/*.ts+(|x)", "*.js", "**/*.js"] }),
+    typescript({
+      include: ["*.ts+(|x)", "**/*.ts+(|x)", "*.js", "**/*.js"],
+      tsconfigOverride: { compilerOptions: { declaration: false } },
```

Comment:
Maybe create a constant (e.g. `const tsconfigOverride = { compilerOptions: { declaration: false } };`), and then use it here as well as in lines 46-48?

Reply:
The Gecko and Browser builds have different `tsconfigOverride` values, so we'd need to modify the value for the browser build anyway.

Reply:
Switched back to the official rollup plugin, which doesn't need us to override tsconfig, so marking this as resolved.